modules/social_features/social_profile/social_profile.module

```diff
@@ -52,6 +52,10 @@ function social_profile_field_widget_form_alter(&$element, FormStateInterface $f
       // @todo Remove this when script for custom selects will be added.
       $element['country_code']['#attributes']['class'][] = 'browser-default';
       break;
+
+    case 'field_profile_summary':
+      $element['value']['#type'] = 'textarea';
```

Comment:
Hi @OH17L is there a reason you chose to do this?
It seems this way it will block our open source site builders to be able to change the actual field type through the UI and instead it will always override it.

Reply:
Thanks for the explanation @OH17L I see what you mean.

Is there an option you prefer out of the 3?

I would imagine it to be the best to use the existing
FieldType: `TextWithSummaryItem` 

Create our own FieldWidget similar to TextareaWithSummaryWidget but additional ensure we can limit the amount of characters.

Reply:
I prefer the 2nd option. We've discussed in the refinement session a case when an existing field can be extended but decided to create a separate field for simply managing access by "Social Profile Fields" and "Social Profile Privacy" modules.

Reply:
I'm also wondering why other widgets (two widgets are here) can be modified in this hook but not mine?

Reply:
Hi @OH17L 

In this case I couldn't see from the commit or inline documentation why this solution was chosen, to try and understand the reason behind it and if that is something I missed or something I should know I asked.

One thing I know from my past bugs is doing it like this breaks sitebuilder capabilities which will result in support request. It's something I rather avoid unless we have a good reason to do otherwise. That is why this question came in to mind. I hope you don't take this feedback personal it's never ever meant like that. 😊